### PR TITLE
Add app.riftshare.RiftShare

### PR DESCRIPTION
**Pull request by achhabra2**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Cross-platform Golang app built using Wails framework. Already on Apple / Windows stores and would love to add this one to Flathub! This would be my first flathub publication so please let me know if I missed something and be gentle :)

### CODE REVIEW

File: app.riftshare.RiftShare.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+app-id: app.riftshare.RiftShare
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
```
Comment (hfiguiere) [inline app.riftshare.RiftShare.yaml:4]: does it really need the GNOME platform?

File: app.riftshare.RiftShare.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        sha256: bc1f55dabc376b9c73d0f0f3386fb48ca5fc9a354eb5d178e1178b30e948ba1e
+        strip-components: 1
+    buildsystem: simple
+    build-commands:
+      - go build -tags desktop,production -ldflags "-w -s" -o RiftShare
+      - install -Dm755 RiftShare /app/bin/RiftShare
+      - install -Dm644 build/linux/app.riftshare.RiftShare.desktop /app/share/applications/app.riftshare.RiftShare.desktop
+      - install -Dm644 build/linux/app.riftshare.RiftShare.png /app/share/icons/hicolor/512x512/apps/app.riftshare.RiftShare.png
+      - install -Dm644 build/linux/app.riftshare.RiftShare.appdata.xml /app/share/metainfo/app.riftshare.RiftShare.appdata.xml
+      - install -Dm644 build/linux/app.riftshare.RiftShare.appdata.xml /app/share/appdata/app.riftshare.RiftShare.appdata.xml
```
Comment (hfiguiere) [inline app.riftshare.RiftShare.yaml:35]:
```suggestion
      - install -Dm644 build/linux/app.riftshare.RiftShare.appdata.xml /app/share/appdata/app.riftshare.RiftShare.appdata.xml
```

you only need it once

**Comment by hfiguiere** [comment]
bot, build app.riftshare.RiftShare

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: app.riftshare.RiftShare.yaml (hunk 1 above)
Comment (achhabra2) [reply app.riftshare.RiftShare.yaml:4]: Yeah unfortunately there is a dependency on libgtk3 and libwebkit for the frontend as required by the framework I'm using.

**Comment by achhabra2** [comment]
bot, build app.riftshare.RiftShare

File: app.riftshare.RiftShare.yaml (hunk 1 above)
Comment (hfiguiere) [reply app.riftshare.RiftShare.yaml:4]: gtk3 is in the Freedesktop-SDK. 

But webkit... indeed. that's probably needed.

**Pull request closed without merging**